frontend/src/common/constants/workers.constants.ts

```diff
@@ -0,0 +1,11 @@
+const WORKERS_PER_PAGE = 5;
+const WORKER_BACK_PAGE = 5;
+const WORKER_NEXT_PAGE = 5;
+const WORKER_CURRENT_PAGE = 1;
```

Comment:
`const WORKER_CURRENT_PAGE = 1;`
I think it's not a const, its state of the current table.